examples/src/content/jcr_root/apps/core-components-examples/clientlibs/clientlib-base-amp/styles/base/logo.less

```diff
@@ -19,7 +19,7 @@
 .cmp-examples-structure__header,
 .cmp-examples-structure__aside {
 
-    .cmp-image__image {
+    .cmp-image__container {
```

Comment:
This was an issue I noticed working on this, but doesn't necessarily have to do with the accordion. The logos that we want to hide on AMP pages were still taking up space (though not rendering) because the entire container needed to be hidden.